pkg/onchain/website/handler.go

```diff
@@ -61,7 +61,7 @@ func RedirectToDefaultResourceInterceptor(req *interceptor.Interceptor) *interce
 
 		splited := removeEmptyStrings(strings.Split(req.Request.URL.Path[len(prefix):], "/"))
 
-		if len(splited) == 1 {
+		if len(splited) == 0 {
```

Comment:
// before we had something like /thyra/websiteUploader, which needs to be converted
		// to an array [websiteUploader] in this specific case the len is 1, and we proceed
		// adding "/.html" to  "websiteuploader"
		// Now i want to keep same functionality but we don't have anymore /thyra, but instead 
		// only /websiteUploader, it will be converted to [] only in this case len is 0,  
		// i can start adding  "/.html"
		// it's working but not simple to understand, i tried to make it simple.  
		// but it's taking more than time allocateed in this specfic PR,
		// i can make it as TODO task : specific PR to refector inly this interceptor

Reply:
i propose you remove the changes in path, to focus only on DNS change in this PR